main/src/io/github/iltotore/iron/constraint/all.scala

```diff
@@ -0,0 +1,6 @@
+package io.github.iltotore.iron.constraint
+
+object all:
```

Comment:
I had to remove `export collection.*` because the three types in there are already being re-exported in the `string` package. To be honest, I don't feel so sure about that re-export. What do you say we get rid of it in `string`?

Reply:
> I had to remove export collection.* because the three types in there are already being re-exported in the string

Does the re-export make some issue or "conflict"? I personally don't want to merge `string` and `collection` together because `string` contains some String-specific constraint like `Match`, `UpperCase` etc...

If there's a conflict, then a better workaround IMO would be to move `MinLength` and `MaxLength` givens in `collection`.

Reply:
> Does the re-export make some issue or "conflict"? 

It doesn't compile 😄 

> I personally don't want to merge string and collection together because string contains some String-specific constraint like Match, UpperCase etc...

I didn't mean to imply that! I think the `string` package should be as it is, but just import the `colllection` types needed instead of doing the re-export of them so we can avoid these conflicts.